api/handler/default.go

```diff
@@ -249,16 +246,23 @@ func (m ArcDefaultHandler) POSTTransactions(ctx echo.Context, params api.POSTTra
 			return ctx.JSON(int(api.ErrStatusBadRequest), e)
 		}
 
-		transactions = make([]interface{}, len(txBody))
+		transactionInputs := make([]*bt.Tx, 0)
```

Comment:
Better would be to initialize the array with a cap, otherwise a malloc is done everytime an element is added:
`transactionInputs := make([]*bt.Tx, 0, len(txBody))`

Reply:
Fixed. Though malloc isn't called on EVERY element. The capacity increases exponentially. But good catch anyway